driver/main_args.ml

```diff
@@ -502,17 +500,12 @@ let mk_unsafe f =
   " Do not compile bounds checking on array and string access"
 ;;
 
-let mk_unsafe_string f =
-  if Config.safe_string then
-    let err () =
-      raise (Arg.Bad "OCaml has been configured with -force-safe-string: \
-                      -unsafe-string is not available")
-    in
-    "-unsafe-string", Arg.Unit err, " (option not available)"
-  else if Config.default_safe_string then
-    "-unsafe-string", Arg.Unit f, " Make strings mutable"
-  else
-    "-unsafe-string", Arg.Unit f, " Make strings mutable (default)"
+let mk_unsafe_string =
+  let err () =
+    raise (Arg.Bad "OCaml has been configured with -force-safe-string: \
+                    -unsafe-string is not available")
```

Comment:
This error message should be changed since there's no longer a way to configure with unsafe strings.

Reply:
Arf I missed that one. Thanks! Fixed.